src/Card/Card.js

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 
 function Card(props) {
-  const userRating = props.displayUserRatings(props.id);
+  const userRating = props.ratings.find(rating => props.id === rating.movieId)
```

Comment:
Was able to refactor DisplayUserReviews, since find will result in falsey was able to get the reviews displaying a bit more dynamically, the original function was also causing the Card and CardComponent tests to fail.